### ISSUE TITLE
Roadmap of detrex

### ISSUE THREAD

**Issue by rentainhe** [IDEA-Research/detrex#89]
We keep this issue open to collect feature requests from users and hear your voice.

You can either:

- Suggest a new feature by leaving a comment.
- Vote for a feature request with 👍. (Remember that developers are sometimes busy and cannot respond to all feature requests, so vote for your most favorable one!)
- Tell us that you would like to help implement new features or review the PRs. (This is the greatest thing to hear about!)

**Comment by caopulan** [comment]
Wandb to log training process and visualize the sampled results.

**Comment by VCBE123** [comment]
support  tensorRT for fast inference

**Comment by wangxuewen99** [comment]
mask-DINO

**Comment by simonJJJ** [comment]
Objects365 dataloader.

**Comment by Hezhexi2002** [comment]
support multiple downstream tasks like keypoint detection or segmention and so on

**Comment by var316** [comment]
Python package for Detrex

**Comment by zldrobit** [comment]
Would you like to upload pre-trained weights on objects365, so we could reproduce the training for COCO dataset? Or we could fine tune Objects365 pre-trained weights on custum datasets.

**Comment by czx94** [comment]
We want ONNX!

**Comment by wangzhaoyang-508** [comment]
hope support salience DETR
